archivist/assets.py

```diff
@@ -46,6 +45,13 @@
 #: be changed.
 DEFAULT_PAGE_SIZE = 500
 
+# These are now hardcoded and not user-selectable. Eventually they will be removed from
+# the backend API and removed from this package.
+BEHAVIOURS = [
```

Comment:
does the `Builtin` behaviour get automatically added in the backend or do we also need to add it here?

Reply:
backend automatically adds it...